support/support.go

```diff
@@ -61,84 +62,141 @@ type Deps struct {
 }
 
 func DeploymentInfo(ctx context.Context, client *codersdk.Client, log slog.Logger) Deployment {
-	var d Deployment
-
-	bi, err := client.BuildInfo(ctx)
-	if err != nil {
-		log.Error(ctx, "fetch build info", slog.Error(err))
-	} else {
+	var (
+		d  Deployment
+		m  sync.Mutex
+		wg sync.WaitGroup
+	)
+
+	wg.Add(1)
```

Comment:
Using [`errgroup.WithContext`](https://pkg.go.dev/golang.org/x/sync/errgroup#WithContext) would be a nice simplification here and allow early exit on error. Simplifies error handling too.

Reply:
Updated!